syntax = "proto3";

message Persons {
    message Person {
        int32 id = 1;
        string first_name = 2;
        string last_name = 3;
        string company_name = 4;
    }
    repeated Person persons = 1;
}

message Empty {
    
}

service PersonService {
    rpc RetreaveAll(Empty) returns (Persons);
}